src/components/AssetInput/MultiAssetPreview/Assets.tsx

```diff
@@ -0,0 +1,44 @@
+import { merge } from "@utilities/merge";
+import React, { FC } from "react";
+import { AssetInputSize } from "../AssetInput";
+import { AssetThumbnail } from "../AssetThumbnail";
+import { MultiAssetPreviewProps } from "./MultiAssetPreview";
+
+export const Assets: FC<Pick<MultiAssetPreviewProps, "assets">> = ({ assets }) => {
```

Comment:
You could also do it the following way, whatever you prefer:
```suggestion
export const Assets: FC<MultiAssetPreviewProps["assets"]>> = ({ assets }) => {
```

Reply:
as discussed on slack. We will leave it like this